src/components/Wheel.jsx

```diff
@@ -0,0 +1,62 @@
+import { Wheel } from 'react-custom-roulette'
+import styled from 'styled-components/macro'
+import prompts from '../data/wheel-prompts.json'
+
+export default function WheelComponent({ winner, mustSpin, setMustSpin }) {
+  const backgroundColors = [
+    '#f94144',
+    '#f3722c',
+    '#f8961e',
+    '#f9c74f',
+    '#90be6d',
+    '#43aa8b',
+    '#577590',
+  ]
```

Comment:
I can not evaluate, if you plan to use these colors in other parts of your app later on, but you could consider to exchange them with css variables in order to make them easier to change/maintain.

Reply:
I don't know yet 🤔 probably not, but if I do, I'll use variables! At least I'm planning on doing so with the other colors as well 😅